src/components/emptyCard/EmptyCard.jsx

```diff
@@ -0,0 +1,39 @@
+import { useLocation, Link } from "react-router-dom";
+import styles from "./emptyCard.module.css";
+
+function EmptyCard() {
+  const location = useLocation();
+  let text = "";
+  switch (location.pathname) {
+    case "/liked":
+      text = "No Liked Videos";
+      break;
+    case "/history":
+      text = "No Videos Watched";
+      break;
+    case "/watchlater":
+      text = "No Videos in Watch Later";
+      break;
+    case "/playlist":
+      text = "No Playlist available";
+      break;
+    default:
+      text = "";
+  }
```

Comment:
appreciation - good that you're showing a single component for all the pages & changing the content based on pathname